Dockerfile

```diff
@@ -36,10 +36,12 @@ RUN set -eux; \
 # https://getcomposer.org/doc/03-cli.md#composer-allow-superuser
 ENV COMPOSER_ALLOW_SUPERUSER=1
 
+ENV PHP_INI_SCAN_DIR="$PHP_INI_DIR/conf.d:$PHP_INI_DIR/app.conf.d"
```

Comment:
```suggestion
ENV PHP_INI_SCAN_DIR=":$PHP_INI_DIR/app.conf.d"
```

Reply:
In this case, PHP is configured with the `--with-config-file-scan-dir=/usr/local/etc/php/conf.d` option, so there is no need to specify this directory manually

Reply:
I tried that, but that didn't worked.  Seems to be working now. I'll apply it